src/pages/SubmissionDetails.js

```diff
@@ -43,6 +43,21 @@ export const SubmissionDetails = ({ submissionId }) => {
       Utils.differenceFromNowInSeconds(submitted)
   }
 
+  const getFilter = filter => {
```

Comment:
Super nit-picky: I would rename this to `applyRunsFilter` since it is actively doing the filtering rather than getting a filter to apply later.

Reply:
this may not be true anymore after Katrina's changes to make this more "functional" - now it's returning a filter function